CHANGELOG.md

```diff
@@ -78,6 +79,67 @@ title: Changelog
+- Fix: allow special characters in uri params [#11788](https://github.com/apache/apisix/pull/11788)
+- Fix: add nil check to conf in body-transformer [#11768](https://github.com/apache/apisix/pull/11768)
+- Fix: use max_req_body_bytes field in custom_format [#11771](https://github.com/apache/apisix/pull/11771)
+- Fix: health checker can't be released due to health parent being released early [#11760](https://github.com/apache/apisix/pull/11760)
+- Fix: use right modifiedIndex for consumer when use credential [#11649](https://github.com/apache/apisix/pull/11649)
+
+### Core
+
+- upgrade openresty version to v1.27.11 [#11936](https://github.com/apache/apisix/pull/11936)
+- add support for "system" ssl_trusted_certificate [#11809](https://github.com/apache/apisix/pull/11809)
+- support _meta.pre_function to execute custom logic before execution of each phase [#11793](https://github.com/apache/apisix/pull/11793)
+- support anonymous consumer [#11917](https://github.com/apache/apisix/pull/11917)
+- accelerate the creation of the consumer cache [#11840](https://github.com/apache/apisix/pull/11840)
+- replace 'string.find' with 'core.string.find' [#11886](https://github.com/apache/apisix/pull/11886)
+- workflow plugin registration [#11832](https://github.com/apache/apisix/pull/11832)
+
+### Plugins
+
+- support embeddings API [#12062](https://github.com/apache/apisix/pull/12062)
+- implement rate limiting based fallback strategy [#12047](https://github.com/apache/apisix/pull/12047)
+- ai-rate-limiting plugin [#12037](https://github.com/apache/apisix/pull/12037)
+- add `valid_issuers` field in `openid-connect` plugin [#12002](https://github.com/apache/apisix/pull/12002)
+- add ai-prompt-guard plugin [#12008](https://github.com/apache/apisix/pull/12008)
+- add jwt audience validator [#11987](https://github.com/apache/apisix/pull/11987)
+- store JWT in the request context [#11675](https://github.com/apache/apisix/pull/11675)
+- support proxying openai compatible LLMs [#12004](https://github.com/apache/apisix/pull/12004)
+- add `ai-proxy-multi` plugin [#11986](https://github.com/apache/apisix/pull/11986)
+- make rate-limit response headers configurable [#11831](https://github.com/apache/apisix/pull/11831)
+- allow workflow config without case [#11787](https://github.com/apache/apisix/pull/11787)
+- support mulipart content-type in `body-transformer` [#11767](https://github.com/apache/apisix/pull/11767)
+- plugins in multi-auth returns error instead of logging it [#11775](https://github.com/apache/apisix/pull/11775)
+- support configuring `key_claim_name` [#11772](https://github.com/apache/apisix/pull/11772)
+- add Total request per second panel in grafana dashboard [#11692](https://github.com/apache/apisix/pull/11692)
+- add ai-rag plugin [#11568](https://github.com/apache/apisix/pull/11568)
+- add ai-content-moderation plugin [#11541](https://github.com/apache/apisix/pull/11541)
```

Comment:
is ai-aws-content-moderation change not part of this release?

Reply:
added